README.md

```diff
@@ -1,22 +1,25 @@
 # shopify-codemod
 
+[![Circle CI](https://circleci.com/gh/Shopify/shopify-codemod.svg?style=svg)](https://circleci.com/gh/Shopify/shopify-codemod)
+
 This repository contains a collection of Codemods written with [JSCodeshift](https://github.com/facebook/jscodeshift) that will help update our old, crusty JavaScript to nice, clean JavaScript.
 
 ## Usage
 
-1. `npm install -g jscodeshift`
-2. `git clone https://github.com/shopify/javascript` or [download the zip file](https://github.com/shopify/javascript/archive/master.zip)
-3. `npm run setup`
-4. `jscodeshift -t <codemod-script> <file>`
-5. Use the `-d` option for a dry-run and use `-p` to print the output for comparison.
+1. `yarn global add jscodeshift`
+2. `git clone https://github.com/shopify/shopify-codemod` or [download the zip file](https://github.com/shopify/shopify-codemod/archive/master.zip)
+3. `cd shopify-codemod`
+4. `yarn install`
+5. `jscodeshift -t <codemod-script> <file>`
+6. Use the `-d` option for a dry-run and use `-p` to print the output for comparison.
 
 ## Documentation
 
 The documentation for each transform can be found in the [docs folder](docs).
 
 ## Contributing
 
-All code is written in ES2015+ in the `transforms/` directory. Make sure to add tests for all new transforms and features. A custom `transforms(fixtureName)` assertion is provided which checks that the passed transformer converts the fixture found at `test/fixtures/{{fixtureName}}.input.js` to the one found at `test/fixtures/{{fixtureName}}.output.js`. You can run `npm test` to run all tests, or `npm run test:watch` to have Mocha watch for changes and re-run the tests.
+All code is written in ES2015+ in the `transforms/` directory. Make sure to add tests for all new transforms and features. A custom `transforms(fixtureName)` assertion is provided which checks that the passed transformer converts the fixture found at `test/fixtures/{{fixtureName}}.input.js` to the one found at `test/fixtures/{{fixtureName}}.output.js`. You can run `yarn test` to run all tests, or `yarn test:watch` to have Mocha watch for changes and re-run the tests.
```

Comment:
Nah, yarn will run scripts if it doesn't find a relevant command.

That can bite in unexpected ways.  `clean` is a common script, but [it'll run a niche yarn builtin instead](https://yarnpkg.com/en/docs/cli/clean).  If we want to avoid that, we should standardize on always having `run` for scripts.